client/src/components/LibraryFolder/LibraryFolder.vue

```diff
@@ -0,0 +1,514 @@
+                    <span class="cursor-pointer" title="Check to select all datasets" @click="toggleSelect"
+                        >&check;</span
+                    >
+                </template>
+                <template v-slot:cell(selected)="{ rowSelected }">
+                    <template v-if="rowSelected">
+                        <span aria-hidden="true">&check;</span>
+                    </template>
+                    <template v-else>
+                        <span class="lib-folder-checkbox" aria-hidden="true">&nbsp;</span>
+                    </template>
+                </template>
+                <!-- Name -->
+                <template v-slot:cell(name)="row">
+                    <div v-if="row.item.editMode">
+                        <textarea
+                            v-if="row.item.isNewFolder"
+                            class="form-control"
+                            name="input_folder_name"
+                            :ref="'name' + row.item.id"
+                            v-model="row.item.name"
+                            rows="3"
+                        ></textarea>
+                        <textarea
+                            v-else
+                            class="form-control"
+                            :ref="'name' + row.item.id"
+                            :value="row.item.name"
+                            rows="3"
+                        ></textarea>
+                    </div>
+                    <div v-else-if="!row.item.deleted">
+                        <a :href="createContentLink(row.item)">{{ row.item.name }}</a>
+                    </div>
+                    <!-- Deleted Item-->
+                    <div v-else>
+                        <div style="color: grey;">{{ row.item.name }}</div>
+                    </div>
+                </template>
+
+                <!-- Description -->
+                <template v-slot:cell(message)="row">
+                    <div v-if="row.item.editMode">
+                        <textarea
+                            v-if="row.item.isNewFolder"
+                            class="form-control input_folder_description"
+                            :ref="'description' + row.item.id"
+                            v-model="row.item.description"
+                            rows="3"
+                        ></textarea>
+                        <textarea
+                            v-else
+                            class="form-control input_folder_description"
+                            :ref="'description' + row.item.id"
+                            :value="row.item.description"
+                            rows="3"
+                        ></textarea>
+                    </div>
+                    <div v-else>
+                        <div class="description-field" v-if="getMessage(row.item)">
+                            <div v-if="getMessage(row.item).length > 40 && !expandedMessage.includes(row.item.id)">
```

Comment:
The numeric value should be part of the `data` function.

Reply:
do you want this check as a separate function?